client/src/components/InputsModal/InputOAuthCredentials.tsx

```diff
@@ -99,7 +99,7 @@ const InputOAuthCredentials: FC<InputOAuthCredentialsProps> = ({
     return (
       <ListItem disabled={field.locked} key={field.name}>
         <TextField
-          disabled={field.locked}
+          disabled={requirement.locked || field.locked}
```

Comment:
Can this just be:

```suggestion
          disabled={requirement.locked}
```

`locked` is never set on any of the fields in `oAuthFields`